whylog/tests/tests_client/test_backtrackerSearcher.py

```diff
@@ -0,0 +1,62 @@
+from unittest import TestCase
+import os.path
+from whylog.tests.tests_client.constants import test_paths, AFewLinesLogParams
+
```

Comment:
separate module imports from standard library imports by a newline, not module imports from each other